sdk/storage/src/blob/blob/mod.rs

```diff
@@ -106,8 +152,12 @@ pub struct BlobProperties {
     #[serde(rename = "Content-Disposition")]
     pub content_disposition: Option<String>,
     #[serde(rename = "Content-MD5")]
+    #[serde(default)]
+    #[serde(with = "md5_optional")]
```

Comment:
More for my education: would `deserialize_with` be more appropriate?

Reply:
Good question... I do not know either 🤷‍♂️. It does not make much difference since the enclosing struct does not implement `Serialize` but your suggestion is more clear so I'll change the code accordingly 👍 .

---
Your question made me realize I have left the previous Deserialize implementation (`impl<'de> Deserialize<'de> for ConsistencyCRC64`) which is no longer used.